addons/medical_damage/functions/fnc_woundsHandlerSQF.sqf

```diff
@@ -122,7 +122,7 @@ private _bodyPartVisParams = [_unit, false, false, false, false]; // params arra
                 _critialDamage = true;
             };
             if ([_unit, _bodyPartNToAdd, _bodyPartDamage, _woundDamage] call FUNC(determineIfFatal)) then {
-                if !(isPlayer _unit && {random 1 > EGVAR(medical,deathChance)}) then {
+                if (!isPlayer || random 1 <= EGVAR(medical,deathChance)) then
```

Comment:
```suggestion
                if (!isPlayer || random 1 < EGVAR(medical,deathChance)) then
```

Very minor thing, but something I alluded to in my other comment. Arma's random operation is exclusive of the upper bound and inclusive of the lower which is 0, so for the setting to be the true chance the comparison should just be less than (consider the edge cases and it'll become clear why).

Reply:
Added missing opening brace too.